// --------------------------------------------------------
// WARNING: XVIZ Protobuf files are unstable, do not use
// --------------------------------------------------------

syntax = "proto3";

import "google/protobuf/descriptor.proto";

extend google.protobuf.MessageOptions {
  string xviz_json_schema = 54200;
}
